eduNEXT/xblock-mindmap
# refactor: update names and messages of xblock fields

### Description
This PR updates the name and messages of the XBlock fields.

### How To Test
1. Install this branch in your environment.
2. Add a block in a unit from Studio.
3. In the edit you should see the new names and messages for the fields.

### Screenshots
![image](https://github.com/eduNEXT/xblock-mindmap/assets/64033729/7d4497db-40fa-4e13-aef4-1a5df9013765)
